client/tests/unit/names-capture.spec.ts

```diff
@@ -32,23 +32,23 @@ describe('send-for-examination.vue', () => {
     })
     it('Initially renders the disabled continue button', () => {
       expect(wrapper.vm.isValid).toBe(false)
-      expect(wrapper.find('#submit-continue-btn').classes().includes('v-btn--disabled')).toBeTruthy()
+      expect(wrapper.find('#submit-continue-btn-false').classes().includes('v-btn--disabled')).toBeTruthy()
     })
     it('demonstrates correct validation logic when designation-1 is entered', async () => {
       store.mutateNameChoices({ key: 'name1', value: 'A Really Nice Name' })
       store.mutateNameChoices({ key: 'designation1', value: 'INC.' })
       await wrapper.vm.$nextTick()
       expect(wrapper.vm.isValid).toBe(true)
-      expect(wrapper.find('#submit-continue-btn-disabled').element).toBeFalsy()
-      expect(wrapper.find('#submit-continue-btn').element).toBeTruthy()
+      expect(wrapper.find('#submit-continue-btn-false').element).toBeFalsy()
+      expect(wrapper.find('#submit-continue-btn-true').element).toBeTruthy()
     })
     it('calls showNextComponent() when continue button is clicked and form is valid', async () => {
       store.mutateNameChoicesToInitialState()
       store.setActiveComponent('NamesCapture')
       store.mutateNameChoices({ key: 'name1', value: 'LALA NAME' })
       store.mutateNameChoices({ key: 'designation1', value: 'INC.' })
       await wrapper.vm.$nextTick()
-      let btn = wrapper.find('#submit-continue-btn')
+      let btn = wrapper.find('#submit-continue-btn-true')
       btn.trigger('click')
       await wrapper.vm.$nextTick()
```

Comment:
You can also do `await Vue.nextTick()`.

Reply:
the official documentation for vue-test-utils consistently uses wrapper.vm.$nextTick(). is there any competing reason to switch aside from maybe slightly less typing?

Reply:
I guess they both work, but I was following this:
https://vue-test-utils.vuejs.org/guides/testing-async-components.html